### PR TITLE
Added location flag to colortool and updated help.

### PR DESCRIPTION
**Pull request by JakeHL**
Also used path combine for schemes directory

### CODE REVIEW

File: tools/ColorTool/ColorTool/Program.cs [hunk 1]
```diff
@@ -304,7 +304,7 @@ private static IntPtr GetStdOutputHandle()
 
         static void PrintSchemes()
         {
-            var schemeDirectory = new FileInfo(new Uri(Assembly.GetEntryAssembly().GetName().CodeBase).AbsolutePath).Directory.FullName + "/schemes"; 
+            var schemeDirectory = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "schemes\\");
```
Comment (oising) [inline tools/ColorTool/ColorTool/Program.cs]: is the trailing `\\` a cross platform issue? Is it better to remove it, or use `System.IO.Path.DirectorySeparatorChar` if it's actually needed?

Comment (DHowett-MSFT) [reply tools/ColorTool/ColorTool/Program.cs]: Actually, yeah -- this is a good question. Using `Path.Combine` followed by manually putting a `\\` on the end seems like an antipattern.

**Comment by JakeHL** [comment]
You're right - this was something I left in by mistake! I'll fix it now.

File: tools/ColorTool/ColorTool/Program.cs [hunk 2]
```diff
@@ -343,6 +343,12 @@ static void PrintSchemes()
             }
         }
 
+        static void PrintSchemesDirectory()
+        {
+            string schemeDirectory = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "schemes\\");
```
Comment (DHowett-MSFT) [inline tools/ColorTool/ColorTool/Program.cs]: (There's still a trailing slash here!)

**Comment by E3V3A** [comment]
@JakeHL .

File: tools/ColorTool/ColorTool/Program.cs (hunk 2 above)
Comment (JakeHL) [reply tools/ColorTool/ColorTool/Program.cs]: Hey! Sorry for the delay on this - been super busy. The issue is all sorted.

**Comment by DHowett-MSFT** [comment]
Thanks!

**Pull request merged**